nats/src/main/proto/com/service/user/user_service.proto

```diff
@@ -0,0 +1,21 @@
+package com.service.user;
+
+import "com/reqreply/user/get_all.proto";
+import "com/reqreply/user/get_by_id.proto";
+import "com/reqreply/user/delete.proto";
+import "com/reqreply/user/edit_by_id.proto";
+
+service UserService {
+
+  rpc getAll(reqreply.user.GetAllUsersRequest)
+      returns (reqreply.user.GetAllUsersResponse);
+
+  rpc getById(reqreply.user.GetByIdUserRequest)
+      returns (reqreply.user.GetByIdUserResponse);
+
+  rpc delete(reqreply.user.DeleteUserRequest)
+      returns (reqreply.user.DeleteUserResponse);
+
+  rpc stream(reqreply.user.StreamRequest)
```

Comment:
and where's stream implementation?

Reply:
Its not the only implementation that was lost. Also adding event type for method subscribeToEvents and some more implementation . Didnt saw that my squash in local git wasn't pushed . Fixed it